server/cmwell-cons/docker.readme

```diff
@@ -0,0 +1,20 @@
+To build a docker image:
+
+1: Use sbt and run packageCmwell
+2: Execute the sollowing command in this directory:
+	docker build -m 4GB --build-arg useAuthorization=false -t <image-name>:latest .
+
+Running the image:
+
+1: docker run -dit -p 8085:9000 --mount type=volume,destination=/home/u/app/cm-well-data,source=cm-well-data  
```

Comment:
regarding the destination - not all of the machines have /home/u, right?

Reply:
On the docker installs, yes, I think so

Reply:
just small comment here - can u please use markup language?put the titles in bold and the steps as list